go.mod

```diff
@@ -22,3 +22,5 @@ require (
 	sigs.k8s.io/kustomize/api v0.5.1
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible
```

Comment:
Any idea why this is required?

Reply:
I added this to resolve following dependency errors:

```
github.com/crossplane-contrib/provider-helm/cmd/provider imports
	k8s.io/client-go/plugin/pkg/client/auth imports
	k8s.io/client-go/plugin/pkg/client/auth/azure imports
	github.com/Azure/go-autorest/autorest: ambiguous import: found package github.com/Azure/go-autorest/autorest in multiple modules:
	github.com/Azure/go-autorest v10.8.1+incompatible (/Users/hasanturken/Go/pkg/mod/github.com/!azure/go-autorest@v10.8.1+incompatible/autorest)
	github.com/Azure/go-autorest/autorest v0.9.0 (/Users/hasanturken/Go/pkg/mod/github.com/!azure/go-autorest/autorest@v0.9.0)
github.com/crossplane-contrib/provider-helm/cmd/provider imports
	k8s.io/client-go/plugin/pkg/client/auth imports
	k8s.io/client-go/plugin/pkg/client/auth/azure imports
	github.com/Azure/go-autorest/autorest/adal: ambiguous import: found package github.com/Azure/go-autorest/autorest/adal in multiple modules:
	github.com/Azure/go-autorest v10.8.1+incompatible (/Users/hasanturken/Go/pkg/mod/github.com/!azure/go-autorest@v10.8.1+incompatible/autorest/adal)
	github.com/Azure/go-autorest/autorest/adal v0.5.0 (/Users/hasanturken/Go/pkg/mod/github.com/!azure/go-autorest/autorest/adal@v0.5.0)
github.com/crossplane-contrib/provider-helm/cmd/provider imports
	k8s.io/client-go/plugin/pkg/client/auth imports
	k8s.io/client-go/plugin/pkg/client/auth/azure imports
	github.com/Azure/go-autorest/autorest/azure: ambiguous import: found package github.com/Azure/go-autorest/autorest/azure in multiple modules:
	github.com/Azure/go-autorest v10.8.1+incompatible (/Users/hasanturken/Go/pkg/mod/github.com/!azure/go-autorest@v10.8.1+incompatible/autorest/azure)
	github.com/Azure/go-autorest/autorest v0.9.0 (/Users/hasanturken/Go/pkg/mod/github.com/!azure/go-autorest/autorest@v0.9.0/azure)
```

Reply:
Oof gotta love k8s imports 🙃